src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/application.ts

```diff
@@ -78,6 +78,12 @@ export const useApplicationStore = defineStore("application", {
       ) {
         this.draftApplication.characterReferences =
           wizardStore.wizardData[wizardStore.wizardConfig.steps.characterReferences.form.inputs.characterReferences.id];
+      } else if (
```

Comment:
Do we just need the condition to check for empty strings? Wondering if we can simplify to

`else { this.draftApplication.characterReferences = []; }`

Reply:
I think we need to check that all fields are empty. Current behavior is that we won't save anything and load up what was previously saved if not all fields are filled. 

If we do else. It would also mean all data is wiped, when the user doesn't input any of the required fields.